Libraries/Image/RCTImageView.h

```diff
@@ -37,4 +37,16 @@ typedef NS_ENUM(NSInteger, UIImageRenderingMode) {
 @property (nonatomic, assign) CGFloat blurRadius;
 @property (nonatomic, assign) RCTResizeMode resizeMode;
 
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+/**
+ * macOS Properties
+ */
+@property (nonatomic, copy) RCTDirectEventBlock onDoubleClick;
```

Comment:
RCTImageView is a category instead of a subclass of RCTView, so we never had these mac/desktop event props on image.   This broke some of the RNTester test pages we never had working -- and would break any App that needed click events on <Image>